test/uk/gov/hmrc/alcoholdutyaccount/models/hods/SubscriptionSummarySpec.scala

```diff
@@ -23,25 +23,29 @@ import uk.gov.hmrc.alcoholdutyaccount.base.SpecBase
 class SubscriptionSummarySpec extends SpecBase {
 
   "SubscriptionSummary" - {
-    "should be able to to be read from a Json" in {
+    "should be able to to be serialised from the full json" in {
 
       val expectedSubscriptionSummary = SubscriptionSummary(
-        typeOfAlcoholApprovedForList = Set(Beer, CiderOrPerry, WineAndOtherFermentedProduct, Spirits),
-        smallCiderFlag = false,
+        typeOfAlcoholApprovedFor = allApprovals,
+        smallciderFlag = false,
         approvalStatus = Approved,
         insolvencyFlag = true
       )
 
       val json =
         """
           |{
-          |    "typeOfAlcoholApprovedForList": [
+          |    "processingDate":"2024-06-11T15:07:47.838Z",
+          |    "organizationName":"Boozy Bob's Beers",
```

Comment:
> There must be no user data that could possibly belong to a real person.
This is a real business. Can you use some ame that doesn't belong to a real person please? (Please go through our Technical Ways of working document as this is a big issue especially in open source projects)

Reply:
Fixed